example/manifest/manifest.go

```diff
@@ -0,0 +1,395 @@
+// A simple example of making a credential manifest and issuing a credential from it, using a credential application.
+// For simplicity the objects are not signed or verified cryptographically.
+// |----------|     |--------------|      |--------|      |--------------|     |---------|      |-------_------|
+// | Verifier  | --> |  Credential  | -->  | Holder | -->  |  Credential  | --> | Verifier | -->  |  Credential  |
+// |          |     |   Manifest   |      |        |      |  Application |     |         |      |   Response   |
+// |----------|     |--------------|      |--------|      |--------------|     |---------|      |--------------|
+package main
+
+import (
+	gocrypto "crypto"
+	"fmt"
+	"time"
+
+	"github.com/TBD54566975/ssi-sdk/credential"
+	"github.com/TBD54566975/ssi-sdk/credential/exchange"
+	"github.com/TBD54566975/ssi-sdk/credential/manifest"
+	"github.com/TBD54566975/ssi-sdk/credential/rendering"
+	"github.com/TBD54566975/ssi-sdk/credential/schema"
+	"github.com/TBD54566975/ssi-sdk/crypto"
+	"github.com/TBD54566975/ssi-sdk/did"
+	"github.com/TBD54566975/ssi-sdk/example"
+	"github.com/TBD54566975/ssi-sdk/util"
+	"github.com/goccy/go-json"
+	"github.com/google/uuid"
+)
+
+// getDIDKey will return a DID key and its private key
+func getDIDKey() (gocrypto.PrivateKey, *did.DIDKey, error) {
+	return did.GenerateDIDKey(crypto.Ed25519)
+}
+
+// Prepare a credential schema that will be issued to issue a credential from a successful Credential Manifest
```

Comment:
nit:
// Prepare a credential schema that will be used to issue a credential from a successful Credential Manifest

Reply:
tyty